package.json

```diff
@@ -34,7 +34,10 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build",
+    "build": "npm run build-js && npm run build-scss && npm run build-css",
```

Comment:
How exactly would the package build work?
I mean previously I just needed to run `npm build`.
Is this still the case?

Reply:
Yes!